pkg/ccl/crosscluster/logical/logical_replication_job_test.go

```diff
@@ -1456,3 +1456,59 @@ func TestShowLogicalReplicationJobs(t *testing.T) {
 	jobutils.WaitForJobToCancel(t, dbA, jobAID)
 	jobutils.WaitForJobToCancel(t, dbA, jobBID)
 }
+
+func TestLogicalReplicationJobDescription(t *testing.T) {
```

Comment:
we spoke offline: i think we decided to delete this test?

Reply:
Yep! I'll push another update shortly